pkg/config/validation/validation_test.go

```diff
@@ -5537,7 +5544,8 @@ func TestValidateAuthorizationPolicy(t *testing.T) {
 					},
 				},
 			},
-			valid: false,
+			valid:   false,
+			Warning: true,
```

Comment:
This shouldn't warn? Its not a DENY rule

Reply:
Yeah, sorry I forgot to add the deny condition while raising the warning. Fixed, thanks.